config/in_character_filter.txt

```diff
@@ -40,3 +42,15 @@ hrp
 coom
 coomer
 bussy
+gock
+cloning
+clone
+sneed
+whitey
+wyci
+nft
+cryptocurrency
+redpill
+blackpill
+bluepill
+thybaron
```

Comment:
wtf did thybaron do to yo dumbass bro???

Reply:
can we add kimitus to the ic filter

Reply:
okay, im making a compromise, we are adding bob newman to the ic filter and that's final.

Reply:
no names of f13 community members in the chat filter........